shipyard/README.md

```diff
@@ -0,0 +1,60 @@
+# Shipyard
+
+## Overview
+
+Shipyard unblocks DevOps teams from bulky, outdated production pipelines. We resolve the pre-production or staging bottleneck with our agile on-demand ephemeral (testing or preview) environments.
+
+The Datadog Shipyard integration automatically passes your [Shipyard][1] ephemeral environment logs to your Datadog dashboard. For every application you have enables the Datadog logs for, Shipyard sends the runtime logs of your application to your dashboard. 
+
+This is useful for: 
+
+- monitoring your application’s runtime status
+- using Datadog to enrich and parse your runtime logs
+- uncover trends and compare the logs from your application’s environments using Datadog’s easy to use interface
+
+## Setup
+
+These instructions assume you have a Shipyard account and application set up. If you do not, you can get started [for free][2]. For more information about setting up the Datadog integration, see the [Shipyard documentation][7]. 
+
+### Configuration
+
+1. Navigate to the [API Keys page][5] and create an API key.
+2. In your Shipyard dashboard, go to **Settings**. 
+3. Under the Datadog section, enter your Datadog API key and the Datadog site URL. *Note: The site URL is not required and therefore we suggest you try your connection with just your API key to begin with*
```

Comment:
```suggestion
3. Under the Datadog section, enter your Datadog API key and the Datadog site URL. 
```
I think it would make it simpler to either require the site URL, or just don't include the optional line above.